INFO - 2023-05-29 03:36:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2023-05-29 03:51:36 --> Class "Config\App\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\Filters\Filters->run('admin/artikel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 4 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab9web\\ci4\\public\\index.php')
CRITICAL - 2023-05-29 03:53:09 --> Class "Config\App\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\Filters\Filters->run('admin/artikel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 4 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab9web\\ci4\\public\\index.php')
INFO - 2023-05-29 03:55:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-29 03:59:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2023-05-29 04:02:01 --> Class "CodeIgniter\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\Filters\Filters->run('admin/artikel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 4 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab9web\\ci4\\public\\index.php')
CRITICAL - 2023-05-29 04:02:31 --> Class "Config\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\Filters\Filters->run('admin/artikel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 4 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab9web\\ci4\\public\\index.php')
CRITICAL - 2023-05-29 04:02:33 --> Class "Config\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\Filters\Filters->run('admin/artikel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 4 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab9web\\ci4\\public\\index.php')
CRITICAL - 2023-05-29 04:03:04 --> Class "Config\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\Filters\Filters->run('admin/artikel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 4 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab9web\\ci4\\public\\index.php')
CRITICAL - 2023-05-29 04:03:20 --> Class "Config\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\Filters\Filters->run('admin/artikel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 4 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab9web\\ci4\\public\\index.php')
INFO - 2023-05-29 04:03:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-29 04:04:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-29 04:04:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-29 04:05:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-29 04:05:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-29 04:06:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-29 04:07:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2023-05-29 04:21:49 --> Maximum execution time of 60 seconds exceeded
in SYSTEMPATH\Database\MySQLi\Connection.php on line 296.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-05-29 04:21:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-29 04:21:53 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\lab9web\ci4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\lab9web\ci4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\lab9web\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\lab9web\ci4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\lab9web\ci4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\lab9web\ci4\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\lab9web\ci4\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\lab9web\ci4\app\Controllers\User.php(31): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\lab9web\ci4\system\CodeIgniter.php(934): App\Controllers\User->login()
#8 C:\xampp\htdocs\lab9web\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\lab9web\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\lab9web\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\lab9web\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\lab9web\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\lab9web\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\lab9web\ci4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\lab9web\ci4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\lab9web\ci4\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\lab9web\ci4\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\lab9web\ci4\app\Controllers\User.php(31): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\lab9web\ci4\system\CodeIgniter.php(934): App\Controllers\User->login()
#7 C:\xampp\htdocs\lab9web\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\lab9web\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\lab9web\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\lab9web\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-05-29 04:21:53 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `useremail` = :useremail:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\User.php(31): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab9web\\ci4\\public\\index.php')
INFO - 2023-05-29 04:21:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2023-05-29 04:21:53 --> Class "Config\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\Filters\Filters->run('admin/artikel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 4 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\lab9web\\ci4\\public\\index.php')
